orchestrator/relayer/src/batch_relaying.rs

```diff
@@ -202,6 +205,7 @@ async fn submit_batches(
                 }
                 let total_cost = total_cost.unwrap();
                 let gas_price_as_f32 = downcast_to_f32(cost.gas_price).unwrap(); // if the total cost isn't greater, this isn't
+                let gas_as_f32 = downcast_to_f32(cost.gas).unwrap(); // same as above re: total cost
```

Comment:
some of those fixed point number types in ethers have multiplication and other operations defined, so perhaps there's no need for this casting to floating points (which could make things behave strangely)